pkg/store/proxy.go

```diff
@@ -73,6 +73,13 @@ type Client interface {
 	Addr() (addr string, isLocalClient bool)
 }
 
+type StoreAPIResult struct {
```

Comment:
We probably don't need this, as we have similar structures elsewhere! Try looking for them!